clients/snowflake/snowflake.go

```diff
@@ -51,6 +52,25 @@ func (s *Store) GetTableConfig(tableData *optimization.TableData) (*types.DwhTab
 	})
 }
 
+func (s *Store) Sweep() error {
+	tcs, err := s.config.TopicConfigs()
+	if err != nil {
+		return err
+	}
+
+	queryFunc := func(dbAndSchemaPair kafkalib.DatabaseSchemaPair) (string, []any) {
+		return fmt.Sprintf(`
+SELECT 
+    table_schema, table_name
+FROM
+    %s.information_schema.tables
+WHERE
+    table_schema = UPPER(?) AND table_name ILIKE ?`, dbAndSchemaPair.Database), []any{dbAndSchemaPair.Schema, "%" + constants.ArtiePrefix + "%"}
```

Comment:
Is it ever possible to have a `table_schema` that is not upper case? Should we do `UPPER(table_schema)`?

Reply:
it is if the schema is escaped, but also no harm in adding UPPER(table_schema)